lib/config-validator.ts

```diff
@@ -134,7 +149,7 @@ type PackageJson = {
         const file = process.env.RENOVATE_CONFIG_FILE ?? 'config.js';
         logger.info(`Validating ${file}`);
         try {
-          await validate(file, fileConfig, strict);
+          await validateAndSetFlag(file, fileConfig, strict);
```

Comment:
```suggestion
          await validateAndSetFlag(file, fileConfig, strict && !!process.env.RENOVATE_CONFIG_FILE);
```

only throw not found if `RENOVATE_CONFIG_FILE` is set, because `config.js` is optional. So it should be possible to run on normal renovate repo and not throw when `config.js` is not found.

Reply:
@liby does this work for you?

Reply:
Apologies for the delay. Yes, the suggested change works well in my local testing. Thanks @viceice for the guidance!

<img width="752" alt="image" src="https://github.com/renovatebot/renovate/assets/38807139/b12d1578-5fe4-4e97-9eac-1d46131e24ce">